R/sf.R

```diff
@@ -453,3 +453,33 @@ st_drop_geometry = function(x) {
 		stop("st_drop_geometry only works with objects of class sf")
 	st_set_geometry(x, NULL)
 }
+
+#' transform method for sf objects
+#' 
+#' Can be used to create or modify variables.
+#' 
+#' @param _data object of class \code{sf}
+#' @param ... Further arguments of the form tag=value
+#'
+#' @export
```

Comment:
`mutate.sf` uses a trick to export it _only_ when `dplyr` is loaded, in order to make `dplyr` not a dependency of `sf`; since `transform` is a method in `base`, this is not needed here.

Reply:
thank you for the clarification. I've already modified it in the code. :smile: